pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -69,6 +94,28 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::weight(10_000)]
+		pub fn update_season_metadata(
+			origin: OriginFor<T>,
+			season_id: SeasonId,
+			metadata: SeasonMetadata,
+		) -> DispatchResult {
+			Self::ensure_organizer(origin)?;
+
+			if season_id >= NextSeasonId::<T>::get() {
+				Err(Error::<T>::UnknownSeason.into())
```

Comment:
Once https://github.com/ajuna-network/Ajuna/pull/52 is merged, we can check on the `Seasons` storage:
```suggestion
			ensure!(Self::seasons(season_id).is_some(), Error::<T>::UnknownSeason);
```